Gundi/tests/Gundi.Tests/Gundi.Tests.csproj

```diff
@@ -8,11 +8,16 @@
         <GenerateProgramFile>false</GenerateProgramFile>
         <WarningsAsErrors>Nullable</WarningsAsErrors>
         <IsPackable>false</IsPackable>
-        <!-- Uncomment to see generated files -->
```

Comment:
Are you sure you want to leave it uncommented ?

Reply:
Yes, but I should uncomment the section about excluding files from the compilator. We can later decide if we want to track changes in generated files.